tests/keras/backend/reference_operations.py

```diff
@@ -118,10 +120,10 @@ def pool(x, pool_size, strides, padding, data_format, pool_mode):
             for (l, l1) in zip(range(pool_size[1]), range(-pool_size[1], 0)):
                 for (m, m1) in zip(range(pool_size[2]), range(-pool_size[2], 0)):
                     y.append(x[:,
-                               :,
-                               k:k1:strides[0],
-                               l:l1:strides[1],
-                               m:m1:strides[2]])
+                             :,
+                             k:k1:strides[0],
+                             l:l1:strides[1],
+                             m:m1:strides[2]])
```

Comment:
There seems to be a hook commit changing the indentation here. Can you revert?

Reply:
I used ctl+alt+l in pycharm to automatically fix formatting - my apologies. I undid the indentation.